### ISSUE TITLE
Update views referenced in context_urls and add regression tests

### ISSUE THREAD

**Issue by jacobwegner** [pinax/pinax-teams#50]

**Comment by jacobwegner** [comment]
Started on https://github.com/pinax/pinax-teams/tree/fix-context-urls

**Comment by jacobwegner** [comment]
Regression tests are going to be a bit tougher to write, due to the`WSGITeamMiddleware` requirement.

Added notes about tests to #49 so this can get unblocked and merged into `master`.
